src/.eslintrc.js

```diff
@@ -1,21 +1,39 @@
 module.exports = {
   overrides: [
     {
-      files: [
-        '!**/*.test.{cjs,js,mjs}',
-        '**/*.{cjs,js,mjs}'
-      ],
+      files: ['**/*.{cjs,js,mjs}'],
+      excludedFiles: ['**/*.test.{cjs,js,mjs}'],
       env: {
         browser: true
       },
       rules: {
-        'no-var': 'off'
+        'no-var': 'off',
+
+        // JSDoc blocks are mandatory
+        'jsdoc/require-jsdoc': [
+          'error', {
+            require: {
+              ClassDeclaration: true,
+              ClassExpression: true,
+              FunctionExpression: true,
+              MethodDefinition: true
+            }
+          }
+        ],
+        'jsdoc/require-param-description': 'warn',
```

Comment:
Looks like the 'warn' value snuck back in or maybe you yere you looking to enable [`require-param-type`](https://github.com/gajus/eslint-plugin-jsdoc/blob/master/.README/rules/require-param-type.md) instead, as that's what we need for typing?

Reply:
@romaricpascal Oh I probably misunderstood, it's been moved from `.eslintrc.js` to `src/.eslintrc.js`

As in JSDoc params (and descriptions) are only needed for our user-facing source code for documentation

Think we should have it optional everywhere?

Thanks

Reply:
I've gone ahead and removed the warnings for `@param` + descriptions

It's something we can monitor during code reviews. Don't want to hassle ongoing spikes with linting issues

(Also set `enableFixer: false` so ESLint `--fix` won't keep adding placeholder `@param` either)